### ISSUE TITLE
script-indent with tabs

### ISSUE THREAD

**Issue by rightaway** [vuejs/eslint-plugin-vue#359]
**Tell us about your environment**

* **ESLint Version:** latest
* **eslint-plugin-vue Version:** latest
* **Node Version:** latest

```js
const vm = new Vue({
	router,
	methods: {
		async myFunction() {
```

The above is in a JS file, and I'm getting `vue/script-indent` errors. The second line says `Expected indentation of 2 tabs but found 1 tab`, and the 4th line says `Expected indentation of 4 tabs but found 2 tabs`. I don't think that section of code should be handled by my eslint rules for plain JS as those rules are working as I'd like for plain JS.

**Comment by mysticatea** [comment]
Thank you for the report.

I think this is a duplicate of #344. Please wait for the next release.
